src/cli/java/org/commcare/util/cli/ApplicationHost.java

```diff
@@ -403,9 +411,34 @@ private void initUser() {
         System.out.println("Setting logged in user to: " + u.getUsername());
     }
 
-    public static void restoreUserToSandbox(UserSandbox sandbox, String username, final String password) {
+    public void restoreUserToSandbox(UserSandbox sandbox, String username,

```

Comment:
Would be nice to break this up into helper methods since this method is ~75 lines long.